src/QueryPressure.WinUI/Common/Observer/Subscription.cs

```diff
@@ -0,0 +1,9 @@
+namespace QueryPressure.WinUI.Common.Observer;
+
+public readonly record struct Subscription(string Key, Action Unsubscribe) : ISubscription
```

Comment:
Is is really any sense in struct here? It seems it is accessed everywhere using interface and will be boxes